File: 3295-report-spam-message/3295-report-spam-message.java
class Solution {
    public boolean reportSpam(String[] message, String[] bannedWords) {
        Set<String> st = new HashSet<>();
        for (String s : bannedWords) {
            st.add(s);
        }
        int count = 0;
        for (String s : message) {
            if (st.contains(s)) {
                count++;
            }
        }
        return count >= 2;
    }
}

File: LeetCode/3295-report-spam-message/3295-report-spam-message.cpp
class Solution {
public:
    bool reportSpam(vector<string>& message, vector<string>& bannedWords) {
        unordered_set<string> st(bannedWords.begin(), bannedWords.end());
        int count = 0;
        for (string &x : message) {
            if (st.find(x) != st.end()) {
                count++;
            }
        }
        return count >= 2;
    }
};